diptest/phase_test.go

```diff
@@ -33,6 +34,7 @@ func (set orderSet) execute(phaseOrdinal int) {
 			phase.Follow("create-order", "Links").Body(map[string]interface{}{
 				"Parts": ord,
 			}).Success()
+			fmt.Printf("o")
```

Comment:
Is this debugging?  It's very minimal for either logging or debugging!

Reply:
No, just some kind of progress indication, since the solo win test is so SLOOOOOOW.